examples/with-particles/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "with-particles",
+  "version": "0.1.0",
```

Comment:
```suggestion
```

These should not be defined in examples. See: https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples